site/zenodo_rdm/moderation/handlers.py

```diff
@@ -77,17 +81,19 @@ def run(self, identity, draft=None, record=None, user=None, uow=None):
                 )
                 return
 
-            score = 0
-            for rule in self.rules:
-                score += rule(identity, draft=draft, record=record)
+            results = {}
+            for name, rule in self.rules.items():
+                results[name] = rule(identity, draft=draft, record=record)
 
             action_ctx = {
                 "user_id": user.id,
                 "record_pid": record.pid.pid_value,
-                "score": score,
+                "results": results,
```

Comment:
minor: there's a part in the `update_moderation_request` task that formats the score/results and includes it in a comment (the cheap way to log results basically)